docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -0,0 +1,378 @@
+{% set short_lang = 'python' %}
+{% if language == 'JavaScript' %}
+    {% set short_lang = 'js' %}
+{% endif %}
+
+{% set lowercase_lang = 'python' %}
+{% if language == 'JavaScript' %}
+    {% set lowercase_lang = 'javascript' %}
+{% endif %}
+
+***********************************************
+Transaction Processor Tutorial  {{ language }}
+***********************************************
+
+Overview
+========
+
+This tutorial covers the creation of a new Sawtooth Lake transaction family in
+{{ language }}, based on the Sawtooth Lake SDK. We will construct a transaction
+handler which implements a distributed version of the multi-player game tic-
+tac-toe
+
+.. note::
+
+    The SDK contains a fully-implemented version of tic-tac-toe. This tutorial
```

Comment:
I would say that there are examples in the sawtooth-core/examples directory, instead of just listing sdk and the path to sdk and leaving it up to the reader to discover that the xo and intkey implementations are in the sdk/examples directory.